frontend/src/features/settings/account/PasswordForm.tsx

```diff
@@ -19,12 +19,14 @@ const PasswordForm = () => {
   const [oldPassword, setOldPassword] = useState('');
   const [password, setPassword] = useState('');
   const [passwordConfirm, setPasswordConfirm] = useState('');
+  const [disabled, setDisabled] = React.useState(false);
```

Comment:
`useState` is imported. See usage on line 21.

Reply:
Thanks, I'll fix that.

Reply:
Fixed by db5c1e6475c264dca426f878eeba2d05971b7f9a.